source/Halibut.Tests/Builders/RedisPendingRequestQueueBuilder.cs

```diff
@@ -65,7 +65,8 @@ public QueueHolder Build()
             var redisTransport = new HalibutRedisTransport(redisFacade);
             var dataStreamStore = new InMemoryStoreDataStreamsForDistributedQueues();
             var messageSerializer = new QueueMessageSerializerBuilder().Build();
-            var messageReaderWriter = new MessageSerialiserAndDataStreamStorage(messageSerializer, dataStreamStore);
+            var baseMessageReaderWriter = new MessageSerialiserAndDataStreamStorage(messageSerializer, dataStreamStore);
+            var messageReaderWriter = new MessageSerialiserAndDataStreamStorageWithExceptionObserver(baseMessageReaderWriter, NoOpMessageSerialiserAndDataStreamStorageExceptionObserver.Instance);
```

Comment:
Just wondering, why do we wrap `baseMessageReaderWriter` in `messageReaderWriter` here? 

We are using the "no op" version of the observer. 

Also, it's a decorator, so we can pass in `baseMessageReaderWriter` as is.

Reply:
Why do I not see this comment on the Files tab, removing the decorator nonsense